parsons/google/google_bigquery.py

```diff
@@ -1136,6 +1109,46 @@ def get_row_count(self, schema: str, table_name: str) -> int:
 
         return result["row_count"][0]
 
+    def get_table_ref(self, table_name):
+        # Helper function to build a TableReference for our table
+        parsed = parse_table_name(table_name)
+        dataset_ref = self.client.dataset(parsed["dataset"])
+        return dataset_ref.table(parsed["table"])
+
+    def _get_job_config_schema(
```

Comment:
moved all the schema logic, including the stuff Austin added a couple PRs ago, into this one function